jenkins/automation/splint-memory-leak-pr.groovy

```diff
@@ -0,0 +1,64 @@
+
+pipeline {
+    agent {
+        node {
+            label "docker-centos-7.9.2009-node"
+        }
+    }
+
+    options {
+        timestamps()
+        buildDiscarder(logRotator(daysToKeepStr: '30', numToKeepStr: '30'))
+        ansiColor('xterm')
+    }
+
```

Comment:
Can we add default parameters here so that this job can also be used instead of - https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/Motr/job/memory-leak-analysis/build?delay=0sec 

So there will be a single job for memory analysis. - 

- It will use PR details when auto-triggered through PR
- It will use developer provided fork repo/branch details when triggered manually.

Reply:
Done